3-platform/warehouse/spec/src/main/java/com/_6core/platform/warehouse/rest/v1/dto/ProductResponseDto.java

```diff
@@ -0,0 +1,12 @@
+package com._6core.platform.warehouse.rest.v1.dto;
+
+import lombok.Data;
+import java.math.BigDecimal;
+
+@Data
+public class ProductResponseDto {
```

Comment:
Let's get rid of `Dto` postfix in our naming. This postfix dont bring any additional information.
```suggestion
public class ProductResponse {
```